md/single-sign-on-with-kerberos.md

```diff
@@ -0,0 +1,331 @@
+# Single sign-on with Kerberos
+
+:::info 
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+This pages explains how to configure your Bonita Platform system to use the Kerberos protocol to provide single sign-on (SSO). It assumes you already have a Correctly configured Windows Domain (AD/KDC/DNS services).
+
+This information applies to a Bonita platform deployed from a bundle (Tomcat or WildFly), not to the Engine launched from Bonita Studio. `<BUNDLE_HOME>` refers to the root directory of the bundle.
+
+Kerberos configuration is at tenant level. Each tenant can use a different authentication method (over Kerberos or not).
+
+:::info 
+**Note:** Bonita uses [Spnego](http://spnego.sourceforge.net/) library that allows to request tickets using the Kerberos protocol.
+:::
+
+## Kerberos single sign-on with Active Directory
+
+Kerberos SSO with AD relies on a minimum on 3 physical machines that are member of the same AD domain:
+
+![Kerberos with AD](images/kerberos-ad.png)<!--{.img-responsive}-->
+
+-	Domain Controller: the AD domain controller providing the SSO tickets through the Kerberos KDC component.
+
+-	Service Server: the server on which resides the service we want to access (Bonita for instance). This server is referred to as the "Principal". It uses the credentials of an AD service account and is member of the domain.
+
+-	Client: the client is the end user’s machine. The client is referred to as the "Subject". It uses the credentials of an AD user account and is member of the domain.
+
+
+## Kerberos overview for Bonita
+
+The target architecture for integrating Kerberos SSO with Bonita and Spnego will involve a custom web application filter: Bonita Kerberos filter.
+
+![Authentication over Kerberos](images/kerberos-overview.png)<!--{.img-responsive}-->
+
+The scenario covered by the filter is the following: 
+
+- The Bonita Kerberos filter intercepts all the requests to bonita portal pages, and checks if the user is already logged in on Bonita 
+    
+    - If already logged in => Allow the access
+    - If not logged in => The request is transferred to the Spnego filter
+
+The Spnego filter will then verify the user’s Kerberos tickets if present or create a new one. After obtaining a valid ticket the filter will store some information (the authenticated user login) in the client request and get back to the Bonita Kerberos filter.
+	
+- The Bonita Kerberos filter will automatically create a Bonita session and let the user through to access the Portal resources.
+	
+:::info 
+**Note:** The Spnego filter is called programmatically by the Bonita Kerberos filter. It is not deployed as a standard web application filter in the application descriptor (web.xml). This is done in order to allow bi-directional exchanges with the Bonita Kerberos filter (regular web filters are only chained in one direction).
```

Comment:
This note can be removed as it was true for the old implementation but it no longer applies